README.md

```diff
@@ -1,81 +1,87 @@
-4. Restore mnemonic wallet by running `yarn restore-mnemonic-wallet`
-5. Make sure your wallet has test ADA for transaction fee
-6. Query tip\
-   `$ cardano-cli query tip (--mainnet | --testnet-magic NATURAL)`
-7. Install packages\
-   `$ yarn install`
-8. Copy .env.example to .end and edit it accordingly\
-   `$ cp .env.example .env`
-9. Create a cip72 metadata file.
-10. Launch **cip72-cli** and follow the instructions\
-    `$ yarn start`
-
-### Restore a wallet using existing keys
+1. Create `phrase.prv` file and fill it up with your mnemonic in space seperated format.
+2. Grant script permission to run `chmod +x ./scripts/restore-mnemonic-wallet.sh`
+3. Restore mnemonic wallet by running `NETWORK=(preview|preprod|mainnet) LOCAL_NODE=(true|false) yarn restore-mnemonic-wallet`
+(Choose appropriate values of the NETWORK and LOCAL_NODE variables)
+
+### b) Restore a wallet using existing keys
 
 If you have your `payment.skey`, `payment.vkey`, `stake.skey` and `stake.vkey`, then you can follow these steps
 
-1. Create a payment address `$ cardano-cli address build --payment-verification-key-file payment.vkey --stake-verification-key-file stake.vkey --out-file payment.addr`
-2. Retrieves the node’s current pool parameters `$ cardano-cli query protocol-parameters --out-file protocol.json (--mainnet | --testnet-magic NATURAL)`
-3. Query tip `$ cardano-cli query tip (--mainnet | --testnet-magic NATURAL)`
-4. Install packages\
-   `$ yarn install`
-5. Copy .env.example to .end and edit it accordingly\
-   `$ cp .env.example .env`
-6. Create a cip72 metadata file.
-7. Launch **cip72-cli** and follow the instructions\
-   `$ yarn start`
+1. Create a payment address
+`$ cardano-cli address build --payment-verification-key-file payment.vkey --stake-verification-key-file stake.vkey --out-file payment.addr`
+2. (LOCAL NODE ONLY) Retrieve the node’s current pool parameters
+`$ cardano-cli query protocol-parameters --out-file protocol.json (--mainnet | --testnet-magic NATURAL)`
+(Choose appropriate network parameter)
 
-### Testnet-magic numbers
+### c) Create a new wallet
 
-- 9: Devnet. `--testnet-magic=9`
-- 2: Preview. `--testnet-magic=2`
-- 1: Preprod. `--testnet-magic=1`
+1. Grant script permission to run `chmod +x ./scripts/setup-new-wallet.sh`
+2. Setup new wallet. `NETWORK=(preview|preprod|mainnet) LOCAL_NODE=(true|false) yarn setup-new-wallet`
+(Choose appropriate values of the NETWORK and LOCAL_NODE variables)
+3. Request Test ADA at [https://docs.cardano.org/cardano-testnet/tools/faucet](https://docs.cardano.org/cardano-testnet/tools/faucet)
+to the generated address from the `payment.addr` file. NB transaction fee it's a variable amount, something around 190.000 lovelace
 
-### Cardano node socket port
+## Step 2: Run the registration script
 
-This script assumes that your `CARDANO_NODE_SOCKET_PORT` is set in your terminal environment. If not, you can set it in `.env` file.
+1. Make sure your wallet has test ADA for transaction fee
```

Comment:
Should we add instructions on how to verify that the wallet has transaction fees?

```suggestion
1. Make sure your wallet has test ADA for transaction fee. Check the useful command section below on a command to query the utxo ).
```